Documentation Migration Preview (#538)
OpenSimulationInterface/open-simulation-interface: doc/architecture/reference_points_coordinate_systems.adoc

> = Reference points and coordinate systems

Reviewer: See PR #516

> With Open Simulation Interface, an object's position coordinates refer to the center of the object's bounding box.

Reviewer 2: ```suggestion
In Open Simulation Interface, object's position is defined by the coordinates of it's center of bounding box.
```

Reviewer: See https://github.com/OpenSimulationInterface/open-simulation-interface/pull/516 for current changes.

> This reference point is not necessarily identical to the origin of the respective coordinate systems.

Reviewer 2: "respective coordinate system" is a bit confusing

Reviewer: Fixed. See https://github.com/OpenSimulationInterface/open-simulation-interface/pull/516 for current changes.
